### DISCUSSION TITLE
How to check the answers of a quiz?

### DISCUSSION THREAD

**Discussion by 10leoman** [microsoft/ML-For-Beginners#138] [Q&A] [answered]
Where do we get the results of the quizzes? Could someone please help me out with this.

**Comment by vishvanathan-k** [comment, marked as answer]
If you have answered wrong, you will get a warning saying 'Try Again', If not then I means it is the correct answer.Once you completed all 3 Qs then you will get you have completed the quiz message.

**Reply by PFCThermalAnalysis** [reply]
Ok, I got it.

**Comment by jlooper** [comment]
The quizzes function as 'warm up' and 'cool down' quizzes to get folks ready to learn and reinforce concepts. They aren't graded.
